content/pages/coc.rst

```diff
@@ -0,0 +1,88 @@
+Code of Conduct
+===============
+
+Pyninsula is dedicated to providing a harassment-free conference experience
+for everyone, regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race,
+age or religion.
+We do not tolerate harassment of conference participants in any form.
+Sexual language and imagery is not appropriate in the meetup,
+including talks.
+Pyninsula participants violating these rules may be sanctioned or
+expelled from the venue at the discretion of the meetup organizers.
+
+Harassment includes, but is not limited to:
+
+* Offensive verbal comments related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, or religion.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following 
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Advocating for, or encouraging, any of the above behaviour
+
+Enforcement
```

Comment:
1. This are what we're "allowed to do", not "how to do it" -- these are rules, not policies. Optimizing policies to avoid retaliation is beyond the scope of this PR.

2. It depends on how general the rules. Some times, we might want to have a private discussion with B if that's feasible. If "A" is actually a set, that diffuses retaliation somewhat.

3. It depends on whether we estimate that this is an "honest mistake" or "abuser". For honest mistakes, hopefully people will appreciate getting the feedback and improve. For abusers, as soon as we see the pattern we should anyway move to banning?